### DISCUSSION TITLE
filling up elasticsearch very quickly

### DISCUSSION THREAD

**Discussion by utmostred** [Security-Onion-Solutions/securityonion#8548] [Unsupported Versions]
Is there a way to filter what is sent to ES?  Don't want ES to capture everything, just the alert events and want to limit the information going in.

**Comment by isaacgolding** [comment]
ES is your central database and yes it will consume a lot of space over time.   But the system is setup to purge older data as you move forward in time.    My drive is 88% full. It has been 88% full for weeks and this is normal.  

Here is a link to the ES logging configuration and other info on ES.  [https://docs.securityonion.net/en/2.3/elasticsearch.html#elasticsearch](https://docs.securityonion.net/en/2.3/elasticsearch.html#elasticsearch)

**Comment by weslambert** [comment]
You can drop events in the ingest node pipeline(s), or by filtering the data from the source application(s) (Ex. Suricata/Zeek).
